config/config_params.go

```diff
@@ -139,6 +139,8 @@ type Config struct {
 	MetadataAddr string `config:"hostname;127.0.0.1;die-on-fail"`
 	MetadataPort int    `config:"int(0,65535);8775;die-on-fail"`
 
+	OpenstackRegion string `config:"string;;local"`
```

Comment:
Bit worried about the prefix since this seem to be being used for non-Openstack data too (host endpoints).  Should we just leave host endpoint status at the old path (or remove it since AIFAIK, it's not used anywhere)?